timelock-impl/src/main/java/com/palantir/atlasdb/timelock/lock/AsyncLockService.java

```diff
@@ -47,6 +48,7 @@
     private final ImmutableTimestampTracker immutableTsTracker;
     private final LeaderClock leaderClock;
     private final LockLog lockLog;
+    private final LockWatchingService lockWatchingService = null;
```

Comment:
Would it be reasonable to return a null object rather than null? i.e. something that does nothing and throws on getWatchState()

Reply:
I didn't bother because there is already a subsequent PR that wires in the impl